src/Commands/ListViewCommand.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace Sven\ArtisanView\Commands;
+
+use Illuminate\Console\Command;
+use Sven\ArtisanView\View;
+
+class ListViewCommand extends Command
+{
+    /**
+    * The name and signature of the console command.
+    *
+    * @var string
+    */
+   protected $signature = 'list:view';
+
+   /**
+    * The console command description.
+    *
+    * @var string
+    */
+   protected $description = 'Prints the directory structure of the views directory';
+
+   /**
+    * Create a new command instance.
+    */
+   public function __construct()
+   {
+       parent::__construct();
+   }
+
+   /**
+    * Execute the console command.
+    *
+    * @return mixed
+    */
+   public function handle()
```

Comment:
I meant like what I did in the other commands. They simply call methods some sort of "helper" class which is tested. The commands themselves are never even touched by tests.